### DISCUSSION TITLE
Launcher crashed right after launch on ARM (Solved)

### DISCUSSION THREAD

**Discussion by TheAlpha313** [legendary-gl/legendary#570] [General]
First of all i would like to thanks a lot for offering of great love and support. This CLI based launcher has given us more than one could imagine, allowed us to run every game possible with epic games but this update have just become a night mirror that i'm extremely concerned about. I have no idea what's the cause of this problem but i can completely guarantee this wasn't on my side, After the update the launcher has completely stopped working. The moment after it's launched it crashes, it was the only hope for mac apple silicon (arm64 based) users to enjoy games from epic games on their VMs and Crossover using our beloved launcher.exe because it didn't only allowed the epic games to run but due to it's nature of being CLI it was lightning fast and took the least amount of computer resources possible as compared to that beefy GUI epic games that was not only just not compatible but also ram, cpu and little bit of gpu intensive. All it was use to require was just copy the launcher.exe file and open it which made it true legend of game launchers but now after trying on multiple windows arm version and also trying to open in crossover it immediately crashes the second after it's launched. I mean come on, by far this was the only hope we had and this update ended that 😔

**Comment by derrod** [comment]
First of all, you can download any older version here: https://github.com/derrod/legendary/releases

Second of all, we'll need a few more details about what is actually happening (or not happening).

**Reply by TheAlpha313** [reply]
Indeed you are right, and that's what i'm sticking with for now, and about knowing the problem yes you are absolutely right that you need more details about what is actually happening. I basically just wanted to address the issue to you so you can try to find and fix. As you know more than 70% of the mac community who can't get their Epic games launcher working rely on CLI interface. Without this they could never have epic and thus one cannot describe the respect for what you are doing ❤️

**Reply by derrod** [reply]
I uploaded a new macOS binary here, please try again with that: https://github.com/derrod/legendary/releases/tag/0.20.33

**Reply by TheAlpha313** [reply]
Hell yeah, you rock bro. Thanks a lot. Works like charm, god bless!
